backend/resolvers/mutations/piece.js

```diff
@@ -0,0 +1,148 @@
+import db from "../../config/sequelize";
```

Comment:
The creation operation for Piece is in this file, as is the creation operation for Portfolio. Portfolio's get created as a side effect of the first piece that goes in them being created.